pkg/ensurance/collector/nodelocal/disk.go

```diff
@@ -0,0 +1,123 @@
+package nodelocal
+
+import (
+	"io/ioutil"
+	"time"
+
+	"github.com/shirou/gopsutil/disk"
+	"k8s.io/klog/v2"
+
+	"github.com/gocrane/crane/pkg/common"
+	"github.com/gocrane/crane/pkg/ensurance/collector/types"
+)
+
+const (
+	diskioCollectorName = "diskio"
+	sysBlockPath        = "/sys/block"
+)
+
+func init() {
+	registerMetrics(diskioCollectorName, []types.MetricName{types.MetricDiskReadKiBPS, types.MetricDiskWriteKiBPS, types.MetricDiskReadIOPS, types.MetricDiskWriteIOPS, types.MetricDiskUtilization}, NewDiskIOCollector)
+}
+
+type DiskTimeStampState struct {
+	stat      disk.IOCountersStat
+	timestamp time.Time
+}
+
+type DiskIOCollector struct {
+	diskStates map[string]DiskTimeStampState
+}
+
+type DiskIOUsage struct {
+	DiskReadKiBps  float64
+	DiskWriteKiBps float64
+	DiskReadIOps   float64
+	DiskWriteIOps  float64
+	Utilization    float64
+}
+
+// NewDiskIOCollector returns a new Collector exposing kernel/system statistics.
+func NewDiskIOCollector(_ *NodeLocalContext) (nodeLocalCollector, error) {
+
+	klog.V(2).Infof("NewDiskIOCollector")
```

Comment:
following code standard, default log level is 2 so V(2) can be removed.
and do we really need this log, maybe it can be removed.

Reply:
I removed it.